### ISSUE TITLE
What is the meaning a TensorGridSurface with no offset_w?

### ISSUE THREAD

**Issue by sean-maptek** [gmggroup/omf-python#114]
The validation accepts the `offset_w` property of a `TensorGridSurface` being undefined or None.

Could you please clarify the meaning of this case? Such that it removes the ambiguity in what software vendors supporting OMF should do in this case.

For me, the most obvious meaning would be the surface is essentially flat and in the plane described by axis_u and axis_v, so it is as-if offset_w is really an array of all zeroes.

**Comment by fwkoch** [comment]
Yes - your interpretation is exactly what I have in mind - no `offset_w` means flat.

I'll update the documentation.

**Comment by dbrookes96** [comment]
Is there a plan to explicitely support `NULL` or empty cells in the omf2 tensor grid surface to support delineation? 

For example if something like `NaN` is written to one or more of the the `offset_w` values.
